core/src/main/scala-2/chisel3/VerificationStatement.scala

```diff
@@ -199,7 +201,7 @@ object stop {
       message.foreach(Printable.checkScope(_))
       when(!Module.reset.asBool) {
         message.foreach(PrintfMacrosCompat.printfWithoutReset(_))
-        pushCommand(Stop(stopId, sourceInfo, Builder.forcedClock.ref, 0))
+        pushCommand(chisel3.internal.firrtl.ir.Stop(stopId, sourceInfo, Builder.forcedClock.ref, 0))
       }
     }
     stopId
```

Comment:
Rather than just moving this file (and `Printf.scala`), I think it would be best to them in the same style as other things even though I know there's not much that can be shared, there are still some common types and utilities.